main.go

```diff
@@ -15,28 +16,51 @@ import (
 	_ "github.com/jackc/pgx/v4/stdlib"
 	"go.uber.org/zap"
 	"golang.org/x/crypto/bcrypt"
+
+	"github.com/ilyakaznacheev/cleanenv"
 )
 
 // Main function
 func main() {
-	//fmt.Println("Test")
+	var env config.Env
+	err := cleanenv.ReadEnv(&env)
+	if err != nil {
+		log.Fatal(fmt.Errorf("read env: %w", err))
+	}
+
+	var config config.Config
+	err = cleanenv.ReadConfig("config.json", &config)
+	if err != nil {
+		log.Fatal(fmt.Errorf("read config: %w", err))
+	}
+
 	logger, _ := zap.NewProduction()
-	dsn := "postgres://postgres:secret@0.0.0.0:5432/test?sslmode=disable&timezone=utc"
-	db, err := sql.Open("pgx", dsn)
+
+	db, err := sql.Open(env.SqlDriver, env.Connection)
```

Comment:
you can't choose it's from underscore imports

Reply:
@igkostyuk oh, unfortunately. So we should stop at postgres for now, we will choose dbName, user, host, port, and params, but not db url

Reply:
i miss something https://pkg.go.dev/github.com/jackc/pgx/v4/pgxpool#ParseConfig looks like dsn not a best name :D

Reply:
> i miss something https://pkg.go.dev/github.com/jackc/pgx/v4/pgxpool#ParseConfig looks like dsn not a best name :D

yep)) I don't know do we need a url or config...